site/content/user-guide/amqp-adapter.md

```diff
@@ -42,6 +42,10 @@ NB: The AMQP adapter needs to be configured for TLS in order to support this mec
 
 After verifying the credentials, the number of existing connections is checked against the configured [resource-limits] ({{< ref "/concepts/resource-limits.md" >}}) by the AMQP adapter.  If the limit is exceeded then the connection request is not accepted.
 
+## Message Limits
+
+Before accepting any telemetry or event messages, the AMQP adapter verifies that the configured [message limit] ({{< ref "/concepts/resource-limits.md" >}}) is not exceeded. The incoming message is declined if the limit is exceeded. 
```

Comment:
please document the corresponding outcome of the transfer for this case in the sections on publishing telemetry/event messages